### PR TITLE
Added support for custom generator function

### PR DESCRIPTION
**Pull request by marucjmar**
Hi, 

I need to generate a specific value for mock. This implementation enables me to use my custom value generator. eg.

config:

```yml
src/test/factory/graphql.ts:
   plugins:
       - add: "import { generateArrayBuffer } from '../generators';"
       - typescript-mock-data:
              typesFile: "../../shared/graphql/generated.ts"
              enumValues: upper-case#upperCase
              typenames: keep
              addTypename: true
              scalars:
                 ArrayBuffer:  generateArrayBuffer()
```

src/test/factory/graphql.ts
```ts
import { generateArrayBuffer } from '../generators;

export const anPakeProtocol = (overrides?: Partial<PakeProtocol>): PakeProtocol => {
    return {
        id: overrides && overrides.hasOwnProperty('id') ? overrides.id! : '1d6a9360-c92b-4660-8e5f-04155047bddc',
        message: overrides && overrides.hasOwnProperty('message') ? overrides.message! : generateArrayBuffer(),
    };
};
```

### CODE REVIEW

**Review by ardeois — COMMENTED** [review]
Great thanks for the PR! 

Could you maybe update the readme?

**Comment by marucjmar** [comment]
@ardeois Done :)

File: README.md [hunk 1]
```diff
@@ -116,6 +116,28 @@ generates:
             AWSTimestamp: unix_time # gets translated to casual.unix_time
 ```
 
+## Scalar custom value generator
```
Comment (ardeois) [inline README.md:119]: Can you move this section inside the `scalars` section?
https://github.com/ardeois/graphql-codegen-typescript-mock-data/blob/579c28a9083e15994f23e6b0c5c7867b949289a2/README.md#scalars--scalar-string-scalardefinition--defaultvalue-undefined

**Review by ardeois — APPROVED** [review]
LGTM, thanks for the updates !

**Comment by ardeois** [comment]
@marucjmar I've just noticed the npm publish failed when I merged.
It's now fixed https://www.npmjs.com/package/graphql-codegen-typescript-mock-data
Your PR is included in version 1.4.0

**Pull request merged**